src/Stryker.CLI/Stryker.CLI.sln

```diff
@@ -17,10 +17,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{7B35D5F5-F983-4034-8A15-952E8C415010}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{7B35D5F5-F983-4034-8A15-952E8C415010}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{7B35D5F5-F983-4034-8A15-952E8C415010}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{7B35D5F5-F983-4034-8A15-952E8C415010}.Release|Any CPU.Build.0 = Release|Any CPU
+		{7B35D5F5-F983-4034-8A15-952E8C415010}.Debug|Any CPU.ActiveCfg = TestProject|Any CPU
```

Comment:
What is this config used for?

Reply:
Actually, it is not used for anything.
I was trying to follow the guide to run the CLI.
```Clone the repository https://github.com/stryker-mutator/stryker-net.git
Open Stryker.CLI.sln
On Stryker.CLI open properties > Debug
Create a new Debug profile
Set Launch as Project
Set WorkingDirectory as your local installation dir, pointing to a UnitTest project example: (C:\Repos\stryker-net\src\Stryker.Core\Stryker.Core.UnitTest)
Run the program with Stryker.CLI as the startup project with the newly created Debug profile```

I should not have committed this.

Reply:
Did you succeed in running the CLI? It looks like you created a SolutionConfiguration instead of a debug configuration.